docker/push_all.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "Push all PyTorch-Ignite docker images"
+
+if [ -z $DOCKER_USER ]; then
+    echo "Can not find DOCKER_USER env variable"
+    echo "Please, export DOCKER_USER=<username> before calling this script"
+    exit 1
+fi
+
+if [ -z $DOCKER_TOKEN ]; then
+    echo "Can not find DOCKER_TOKEN env variable"
+    echo "Please, export DOCKER_TOKEN=<token> before calling this script"
+    exit 1
+fi
+
+set -eu
+
+echo $DOCKER_TOKEN | docker login --username=$DOCKER_USER --password-stdin
+
+set -xeu
+
+image_name="base"
+image_tag=`docker run --rm -it pytorchignite/${image_name}:latest -c 'python -c "import torch; import ignite; print(torch.__version__ + \"-\" + ignite.__version__, end=\"\")"'`
+
+for image_name in "base" "vision" "nlp" "apex" "apex-vision" "apex-nlp"
+do
+
+    docker push pytorchignite/${image_name}:latest
+    docker push pytorchignite/${image_name}:${image_tag}
+
+done
+
+
+image_name="hvd-base"
+image_tag=`docker run --rm -it pytorchignite/${image_name}:latest -c 'python -c "import torch; import ignite; print(torch.__version__ + \"-\" + ignite.__version__, end=\"\")"'`
+
+for image_name in "hvd-base" "hvd-vision" "hvd-nlp" "hvd-apex" "hvd-apex-vision" "hvd-apex-nlp"
+do
+
+    docker push pytorchignite/${image_name}:latest
+    docker push pytorchignite/${image_name}:${image_tag}
+
+done
+
+image_name="msdp-apex-base"
+image_tag=`docker run --rm -it pytorchignite/${image_name}:latest -c 'python -c "import torch; import ignite; print(torch.__version__ + \"-\" + ignite.__version__, end=\"\")"'`
+
+for image_name in "msdp-apex-base" "msdp-apex-vision" "msdp-apex-nlp"
+do
+
+    docker push pytorchignite/${image_name}:latest
+    docker push pytorchignite/${image_name}:${image_tag}
+
+done
```

Comment:
Once pushed, the local repository would maybe need a `clean` step. It also seems that multi-stage builds generate, `<none>` images:
```bash
docker images | grep 'pytorchignite\|<none>' | awk '{print $3}' | xargs docker rmi -f
```
what do you think ?

Reply:
Yes, this is important for a local system, on the CI it is less important I think.